ibis/backends/duckdb/__init__.py

```diff
@@ -40,6 +38,38 @@
     import torch
 
 
+class _SettingsPassThrough:
```

Comment:
Can we call this `Settings` (or `_Settings`) and stick it near the base classes?